sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -418,3 +425,166 @@ def analyze_sentiment(  # type: ignore
+        :type top: int
+        :param show_stats: (Optional) if set to true, response will contain input and document level
+         statistics.
+        :type show_stats: bool
+        :keyword callable cls: A custom type or function that will be passed the direct response
+        :return: An iterator like instance of either HealthJobTaskResponse or the result of cls(response)
+        :rtype: ~azure.core.paging.ItemPaged[~azure.ai.textanalytics.HealthJobTaskResponse]
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+        if self._api_version != ApiVersion.V3_2_preview_1:
+            raise NotImplementedError(
+                "The endpoint get_healthcare_results is only available for v3.2-preview.1"
+            )
+
+        try:
+            return self._client.get_healthcare_results(  # type: ignore
+                job_id,
+                skip,
+                top,
+                show_stats,
+                **kwargs
+            )
+        except HttpResponseError as error:
+            process_batch_error(error)
+
+    @distributed_trace
+    def begin_health(  # type: ignore
+        self,
+        source_url,  # type: str
+        model_version=None,  # type: Optional[str]
+        storage_source=None,  # type: Optional[str]
+        **kwargs  # type: Any
+    ):
+        # type: (...) -> LROPoller[None]
+        """Health.
+
+        The API returns a list of recognized healthcare entities and relations for each valid document.
+
+        :param source_url: The container URI where inputs are stored.
+        :type source_url: str
+        :param model_version: (Optional) This value indicates which model will be used for scoring. If
+         a model-version is not specified, the API should default to the latest, non-preview version.
+        :type model_version: str
+        :param storage_source: The prefix for the blob folder container inputs, if applicable.
+        :type storage_source: str
+        :keyword callable cls: A custom type or function that will be passed the direct response
+        :keyword str continuation_token: A continuation token to restart a poller from a saved state.
+        :keyword polling: True for ARMPolling, False for no polling, or a
+         polling object for personal polling strategy
+        :paramtype polling: bool or ~azure.core.polling.PollingMethod
+        :keyword int polling_interval: Default waiting time between two polls for LRO operations if
+         no Retry-After header is present.
+        :return: An instance of LROPoller that returns either None or the result of cls(response)
+        :rtype: ~azure.core.polling.LROPoller[None]
+        :raises ~azure.core.exceptions.HttpResponseError:
+        """
+
+        if self._api_version != ApiVersion.V3_2_preview_1:
+            raise NotImplementedError(
+                "The endpoint begin_health is only available for v3.2-preview.1"
+            )
+
+        string_index_type = "unicodecodepoint"
```

Comment:
I didn't expose `string_index_type` since we had talked about only using unicode for python